otsdaq/SOAPUtilities/SOAPMessenger.cc

```diff
@@ -91,7 +91,7 @@ xoap::MessageReference SOAPMessenger::sendWithSOAPReply(XDAQ_CONST_CALL xdaq::Ap
 	try
 	{
 		message->getMimeHeaders()->setHeader("Content-Location", d->getURN());
-		message->getMimeHeaders()->setHeader("xdaq-pthttp-connection-timeout", "30" /*seconds*/);
+		message->getMimeHeaders()->setHeader("xdaq-pthttp-connection-timeout", "300" /*seconds*/);
```

Comment:
5 minutes. During Mu2e Global Run 1 experience, 2 minutes would not be enough for artdaq interface configure transition code blocks.